components/TestCases.jsx

```diff
@@ -1,5 +1,4 @@
-import { useState, useContext } from "react";
-import CodeContext from "../components/CodeContext";
+import { useState } from "react";
 
 const TestCaseInput = ({ tests, state }) => {
   return (
```

Comment:
add a check here to make sure tests exist before rendering it, else just render an empty div; something along the lines of `{tests ? /* all that stuff */ : <div />}`